instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -200,13 +203,31 @@ def _traced_execute_pipeline(func, instance, args, kwargs):
                 response_hook(span, instance, response)
             return response
 
+    def _traced_get_connection(func, connection_pool, command_name, *keys, **options):
+        span_name = "random-test"
+        with tracer.start_as_current_span(
+            span_name, kind=trace.SpanKind.CLIENT
+        ) as span:
+            response = func(command_name, *keys, **options)
+            metric_labels = {
+                SpanAttributes.DB_CLIENT_CONNECTIONS_USAGE: connection_pool._created_connections,
```

Comment:
Does this `SpanAttributes.DB_CLIENT_CONNECTIONS_USAGE`? The metric name is not same as span attributes. The list of attributes to add for these measurements are state of the connection (`idle` or `used`) and the pool name

Reply:
Hi @srikanthccv, actually the ConnectionPool does not include a name that we can use on the metric, does this look a bit better? I was thinking we could use the PID to identify the connection pool:

```
{
        "db.client.connection.usage.state": "used|idle",
        "db.client.connection.usage.name": instance.connection_pool.pid,
}
```